test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,73 @@
+require "test_helper"
+
+describe UsersController do
+
+  describe "index" do
+    it "can get the index path" do
+      # Act
+      get users_path
+      # Assert
+      must_respond_with :success
+    end
+  end
+
+  describe "new" do
+    it "can get the new user" do
+
+      # Act
+      get new_user_path
+
+      # Assert
+      must_respond_with :success
+    end
+  end
+
+  describe "show" do
+    it "can get a valid user" do
+          # Act
+          get user_path(first.id)
+
+      # Assert
+      must_respond_with :success
+    end
+
+    it "will show flash and render status for invalid id" do
+      # Act
+      get user_path(-1)
+
+      # Assert
+      must_respond_with :render
```

Comment:
Render is not a response code.

```suggestion
      must_respond_with :redirect
```